src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -1111,6 +1204,16 @@ void GenerateMessageDocComment(io::Printer* printer,
     "fullname", EscapePhpdoc(message->full_name()));
 }
 
+void GenerateServiceDocComment(io::Printer* printer,
+                               const ServiceDescriptor* service) {
+  printer->Print("/**\n");
+  GenerateDocCommentBody(printer, service);
+  printer->Print(
+    " * Protobuf type <code>^fullname^</code>\n"
+    " */\n",
+    "fullname", EscapePhpdoc(service->full_name()));
```

Comment:
This will need to be added to https://github.com/google/protobuf/pull/3131, or vice-versa depending on which is merged first